lua/nvim-treesitter/textobjects/attach.lua

```diff
@@ -10,10 +10,24 @@ function M.make_attach(normal_mode_functions, submodule)
     lang = lang or parsers.get_buf_lang(bufnr)
 
     for _, function_call in pairs(normal_mode_functions) do
+      local description
+      if type(function_call) == "table" then
+        function_call, description = unpack(function_call)
```

Comment:
is the function call an "array" table ? If it is a record like table this will not work.

Reply:
I think the intention is to have a record like table, with a query and a description keys

Reply:
It's not used at the moment, I intended to have `{function, description}`. It is not used atm. Better to remove?

Reply:
I removed this feature which was intended for internal documentation of the functions.

Reply:
But why? I thought the entire PR was to allow end users to customize the description of the created keybindings.

Reply:
You don't understand. There were three sources for descriptions 1) auto-generated, 2) set by the plugin 3) set by the user. It would always prefer the higher number. 2) didn't exist. So it's 1) auto-generated, 2) set by the user